oraichain/assetlist.json

```diff
@@ -75,6 +75,7 @@
       "name": "ATOM",
       "display": "atom",
       "symbol": "ATOM",
+      "coingecko_id": "cosmos",
```

Comment:
Please remove all these coingecko_ids; they should only be defined where the asset is natively issued.